architecture.md

```diff
@@ -0,0 +1,55 @@
+# 🧱 Architecture (WIP)
+
+Although the merge NFT collection subgraph is made specifically for that collection. It is a good exercise to think about other NFT collections and if there is a common architecture that satisfies most of them. Stakeholders need to be identified first before creating architecture that satisfies their individual needs:
+
+- Subgraph developers (usually developers within the NFT core team)
+- Data analysts (crypto data companies; ex.) Messari, trad-fi companies, NFT dashboards, etc.)
+- Indexers
+- Curators
+- The Graph Foundation / Edge and Node / other teams
+
+Some assumptions for this base NFT subgraph:
+
+- Code will live within this squirtle dev repo for now.
+- Documentation will be outlined for the base schema.
+
+## 📄 Base Schema
+
+Now we consider what the base schema will look like.
+
+Some basic guidelines:
+
+- It should be simple enough for a traditional finance data scientist to read the schema and query it and fully understand what is going on. No blockchain experience required.
+- But it should be descriptive enough so that someone can get useful information out of it. Hence why it revolves around NFTs, and we need to standardize NFT nomenclature.
+
+This is just a starting point. I have written out the architectural decisions for the pak merge collection as an example below:
+
+## ⚫️ Pak merge. Collection Subgraph Schema
+
+Let's think about this in databases, and how they may connect with one another. As well let's identify which stakeholders would use these databases.
```

Comment:
thinking about it in "databases" is wrong. a subgraph is a single postgres database. 

within a postgres db there are tables, and within tables, there are rows and columns. but that is not neceassary to know for you. fyi, this is about the full extent of how much I understand postgres, lol

Reply:
Modified the wording to reflect your clarification here. Thanks!